vendor/github.com/moby/sys/mountinfo/mounted_unix.go

```diff
@@ -33,13 +32,13 @@ func mountedByStat(path string) (bool, error) {
 
 func normalizePath(path string) (realPath string, err error) {
```

Comment:
Why was this change required for this PR? if we're relying on the error returned from this library not being wrapped, that seems really fragile. Shouldn't we be using errors.As to get the ErrNotExist error if present?

Reply:
(I don't object to picking up the dependency bump otherwise)

Reply:
So the suggestion is to not bump the dependency here, instead fixing the tests where it checks notfound error to use error.As, right?

Reply:
This change is required because k8s depends on os.IsExist and os.IsNotExist: https://github.com/kubernetes/kubernetes/blob/1c526a536f83ca7a0dd77f455f85c336bca7e801/pkg/volume/util/subpath/subpath_linux.go#L103-L107 instead of errors.Is(err, fs.ErrNotExist) && !errors.Is(err, fs.ErrNotExist)

We can modify the PR to include this change as well or do as a follow up.

Reply:
> The function is documented as returning ErrNotExist. 

Are you referring function IsMountPoint() here which changed to add [MountedFast](https://github.com/moby/sys/blob/main/mountinfo/mounted_linux.go#L30) from the library, right? Yes, I think we can unwrap error there

Reply:
> Are you referring function IsMountPoint() here which changed to add [MountedFast](https://github.com/moby/sys/blob/main/mountinfo/mounted_linux.go#L30) from the library, right? Yes, I think we can unwrap error there

if we're unwrapping there (and testing to verify unwrapping is working), do we still need this library bump? if not, can we drop the dependency change from this PR to make it clear that this PR doesn't depend on the `github.com/moby/sys/mountinfo` library returning unwrapped errors?